inc/MicroBitConfig.h

```diff
@@ -292,3 +292,17 @@
 #endif
 
 #endif
+
+// Defines default behaviour of triple-tap-reset-to-pair feature.
+// 0: Feature disabled
+// 1: Feature enabled. Triple tap of reset button enters Bluetooth pairing mode.
+#ifndef MICROBIT_TRIPLE_RESET_TO_PAIR
+    #define MICROBIT_TRIPLE_RESET_TO_PAIR                 1
+#endif
```

Comment:
Renamed this from `DEVICE_TRIPLE_RESET_TO_PAIR` to `MICROBIT_TRIPLE_RESET_TO_PAIR` to follow the current pattern.
Did a GitHub search and there is no usage of this flag outside of this repo (and its forks).